ocrd/ocrd/workspace.py

```diff
@@ -69,20 +70,25 @@ class Workspace():
         baseurl (string) : Base URL to prefix to relative URL.
     """
 
-    def __init__(self, resolver, directory, mets=None, mets_basename='mets.xml', automatic_backup=False, baseurl=None):
+    def __init__(self, resolver, directory, mets=None, mets_basename='mets.xml', automatic_backup=False, baseurl=None, mets_server_host=None, mets_server_port=None, mets_server_socket=None):
         self.resolver = resolver
         self.directory = directory
         self.mets_target = str(Path(directory, mets_basename))
         self.overwrite_mode = False
+        self.is_remote = bool(mets_server_host or mets_server_socket)
         if mets is None:
-            mets = OcrdMets(filename=self.mets_target)
+            if self.is_remote:
+                mets = ClientSideOcrdMets(mets_server_host, mets_server_port, mets_server_socket)
+            else:
+                mets = OcrdMets(filename=self.mets_target)
         self.mets = mets
         self.automatic_backup = automatic_backup
         self.baseurl = baseurl
         #  print(mets.to_xml(xmllint=True).decode('utf-8'))
 
     def __str__(self):
-        return 'Workspace[directory=%s, baseurl=%s, file_groups=%s, files=%s]' % (
+        return 'Workspace[remote=%s, directory=%s, baseurl=%s, file_groups=%s, files=%s]' % (
+            not not self.is_remote,
```

Comment:
```suggestion
            self.is_remote,
```
not needed any more because of previous refactoring